test-projects/gts/.eslintrc.cjs

```diff
@@ -4,7 +4,12 @@ module.exports = {
   root: true,
   parserOptions: {
     project: true,
-    tsconfigRootDir: __dirname
+    tsconfigRootDir: __dirname,
+    ember: {
```

Comment:
proposed way to opt in to extension checking (which would be slower)

Reply:
I think it would be enough if we only to extension checking if type aware linting is enabled. So checking project: true should be enough?

Reply:
maybe we can "do it automatically" when an import doesn't have an extension? and we can get rid of this config?